frontend/src/view/views/Maps.vue

```diff
@@ -1,25 +1,66 @@
-          <template v-else="" v-for="(map, idx) in maps">
-            <div class="zone-map-container" v-if="map.weight > 0" :key="`container-${idx}`">
-              <div class="zone-map-header" :key="`header-${idx}`">
-                <div class="zone-map-name" :key="`name-${idx}`" v-html="map.name"></div>
-                <div class="zone-map-weight" :key="`weight-${idx}`">{{ map.weight }}</div>
+  <div class="zone-maps">
+    <div class="zone-title" v-t="'NCZONE_MAPS'"></div>
+    <div class="zone-content">
+      <div class="loading" v-if="loading"><span v-t="'NCZONE_LOADING'"></span></div>
+      <div class="error" v-else-if="error"><span v-t="'NCZONE_ERROR_LOADING'"></span></div>
+      <template v-else="" v-for="(map, idx) in maps">
+        <div class="zone-map-container" v-if="map.weight > 0" :key="`container-${idx}`">
+          <div class="zone-map-header" :key="`header-${idx}`">
+            <div class="zone-map-arrow" :key="`arrow-${idx}`" @click="{{ toggleMap(map.id) }}">
+              <span class="fa fa-angle-right" v-if="mapInfo.id !== map.id"></span>
+              <span class="fa fa-angle-down" v-if="mapInfo.id === map.id"></span>
+            </div>
+            <div class="zone-map-name" :key="`name-${idx}`" v-html="map.name"></div>
+            <div class="zone-map-weight" :key="`weight-${idx}`">{{ map.weight }}</div>
+          </div>
+          <div class="zone-map-civs-info" v-if="mapInfo.id === map.id">
+            <div v-if="!loadMap">
+              <div class="zone-map-description" @dblclick="{{ editDescription() }}">
+                <div v-if="map.description || canEditMapDescription">
+                  <vue-markdown v-if="!editDescr" class="zone-map-description-text">{{ map.description ? map.description : '*' + $t('NCZONE_EMPTY_DESCRIPTION') + '*' }}</vue-markdown>
+                  <div v-else>
+                    <textarea v-model="tempDescription" rows="10"></textarea><br />
+                    <button @click="{{ saveDescription() }}">{{ $t('NCZONE_SAVE') }}</button>
+                  </div>
+                </div>
+                <div class="zone-map-description-image">
+                </div>
+              </div>
+              <div class="zone-map-civs-table">
+                <div class="zone-map-civs-table-head">{{ $t('NCZONE_CIV') }}</div>
+                <div class="zone-map-civs-table-head">{{ $t('NCZONE_MULTIPLIER') }}</div>
+                <div class="zone-map-civs-table-head">{{ $t('NCZONE_FORCE_DRAW') }}</div>
+                <div class="zone-map-civs-table-head">{{ $t('NCZONE_PREVENT_DRAW') }}</div>
+                <div class="zone-map-civs-table-head">{{ $t('NCZONE_BOTH_TEAMS') }}</div>
+              <template v-for="(civ, idy) in mapInfo.civInfo">
+                <div class="zone-map-civs-table-name" :class="{'even': idy % 2 === 0, 'odd': idy % 2 !== 0}" :key="`name-${idy}`">{{ $t(civ.civ_name) }}</div>
```

Comment:
denke es ist besser, den header in ein div und jede row auch in ein div zu packen und dann mit css nth-child oder sowas auf odd/even zu reagieren, statt hier odd/even klassen ranzubauen

Reply:
:heavy_check_mark: